spec/graphql/language/printer_spec.rb

```diff
@@ -0,0 +1,203 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe GraphQL::Language::Printer do
```

Comment:
Is this ported from `generation_spec.rb`? If so, do you think we can remove that file without losing any coverage? (If so, :+1: from me, so don't leave dead code around.)

Reply:
this is ported from generation_spec yes! I think coverage is the same with or without this test. I was keeping this one because I was scared of forgetting to test the printer if generation is ever deprecated / 🔥 🤔

Reply:
Sorry, it seems like I didn't express myself well 😬 

I think we should _keep_ `printer_spec.rb`, since a lot of logic was moved into `printer.rb`. 

I also think we should _reduce_ `generation_spec.rb`, since a lot of logic was taken _out_ of `generation.rb`. Maybe it just needs one test, that it uses the passed-in printer, but to be honest, it could probably just be removed entirely. 

(I agree with 

> scared of forgetting to test

!)

Reply:
Ah I see what you mean now! Agreed!